src/http/h1/parse.rs

```diff
@@ -114,12 +114,7 @@ impl Http1Transaction for ServerTransaction {
         let init_cap = 30 + head.headers.len() * AVERAGE_HEADER_SIZE;
         dst.reserve(init_cap);
         debug!("writing headers = {:?}", head.headers);
-        if head.version == ::HttpVersion::Http11 && head.subject == ::StatusCode::Ok {
-            extend(dst, b"HTTP/1.1 200 OK\r\n");
-            let _ = write!(FastWrite(dst), "{}", head.headers);
-        } else {
-            let _ = write!(FastWrite(dst), "{} {}\r\n{}", head.version, head.subject, head.headers);
-        }
+        let _ = write!(FastWrite(dst), "{} {}\r\n{}", head.version, head.subject, head.headers);
```

Comment:
Lol, I never want to me in charge of performance sensitive code.

I'll add a comment instead. Similar to the comment for date, do you remember what that speedup was?